edb/schema/indexes.py

```diff
@@ -244,16 +244,10 @@ def get_root(
         self,
         schema: s_schema.Schema,
     ) -> Index:
-        if not self.get_abstract(schema):
-            name = sn.shortname_from_fullname(self.get_name(schema))
-            index = schema.get(name, type=Index)
```

Comment:
Is this a fix for an unrelated bug? Abstract indexes not finding root correctly?